Games/Niko: Through The Dream/Steam/script.json

```diff
@@ -0,0 +1,7 @@
+{
+	"scriptName": "Steam",
+	"compatibleOperatingSystems": ["MACOSX", "LINUX"],
+	"testingOperatingSystems": ["LINUX"],
```

Comment:
It should be set if you are unsure that the script will run properly for everybody (like if it requires a huge amount of hack, or if it is not compatible with a any graphics cards, etc...)